platform-sdk/consensus-otter-tests/build.gradle.kts

```diff
@@ -35,17 +36,51 @@ testModuleInfo {
     requires("com.swirlds.metrics.api")
     requires("org.hiero.consensus.utility")
     requires("org.apache.logging.log4j")
-    requires("awaitility")
-    runtimeOnly("io.grpc.netty.shaded")
+}
+
+testing.suites {
+    val testOtter by
+        registering(JvmTestSuite::class) {
+            useJUnitJupiter()
+
+            dependencies {
+                implementation(project())
+                implementation(project.dependencies.testFixtures(project()))
+                implementation(project(":swirlds-common"))
+                implementation(project(":swirlds-platform-core"))
+                implementation("org.assertj:assertj-core")
+                implementation("org.junit.jupiter:junit-jupiter-params")
+                implementation("com.github.spotbugs:spotbugs-annotations")
+                runtimeOnly("io.grpc:grpc-netty-shaded")
+            }
+
+            targets {
+                all {
+                    testTask.configure {
+                        // Disable all parallelism
+                        systemProperty("junit.jupiter.execution.parallel.enabled", false)
+                        systemProperty(
+                            "junit.jupiter.testclass.order.default",
+                            "org.junit.jupiter.api.ClassOrderer\$OrderAnnotation",
+                        )
```

Comment:
This is not necessary. I don't see `OrderAnnotation` used in the test code. And I would highly recommend to avoid depending on test order.
If I miss something and this is needed, it would be good to add an explanation about why it is needed.